src/plugin/protocol-httpclient/src/java/org/apache/nutch/protocol/httpclient/HttpFormAuthentication.java

```diff
@@ -135,6 +143,26 @@ public boolean getFollowRedirects() {
       }
     }
   }
+  
+  /**
+   * @throws NoSuchFieldException
```

Comment:
Sure. I have updated this source and re-pushed.
And, add an updated httpclient-auth.xml.template for how to configure this cookie policy.